internal/db/schema/migrations/postgres/14/01_event.up.sql

```diff
@@ -0,0 +1,250 @@
+                'every',
+                'error',
+                'audit',
+                'observation',
+                'system'
+            )
+        )
+);
+comment on table event_type_enm is
+'event_type_enm is an enumeration table for the valid event types within '
+'the domain';
+
+create table event_format_type_enm (
+    name text primary key
+        constraint only_predefined_event_format_types_allowed
+        check (
+            name in (
+                'cloudevents-json',
+                'cloudevents-text',
+                'hclog-text',
+                'hclog-json'
+            )
+        )
+);
+comment on table event_format_type_enm is
+'event_format_type_enm is an enumeration table for the valid event format types '
+'within the domain';
+
+create table event_config (
+    public_id wt_public_id primary key,
+    scope_id wt_scope_id not null
+        constraint iam_scope_global_fkey
+            references iam_scope_global(scope_id)
+            on delete cascade
+            on update cascade,
+        constraint scope_id_uq -- only allow one config per scope
+            unique (scope_id),
+    name wt_name,
+    description wt_description,
+    create_time wt_timestamp,
+    update_time wt_timestamp,
+    version wt_version
+);
+comment on table event_config is
+'event_config is a table where each row defines the event configuration for '
+'a scope.  Currently, the only supported scope is global';
+
+create trigger
+    update_version_column
+after update on event_config
+    for each row execute procedure update_version_column();
+
+create trigger
+    update_time_column
+before update on event_config
+    for each row execute procedure update_time_column();
+
+create trigger
+    default_create_time_column
+before insert on event_config
+    for each row execute procedure default_create_time();
+
+create table event_type_enabled (
+    config_id wt_public_id
+        constraint event_config_fkey
+            references event_config(public_id)
+            on delete cascade
+            on update cascade,
+    event_type text not null
+        constraint event_type_enm_fkey
+            references event_type_enm (name)
+            on delete restrict
+            on update cascade,
+    primary key (config_id, event_type)
+);
+comment on table event_type_enabled is
+'event_type_enabled is a table where each row represents that eventing has '
+'been enabled for the specified event type in an event configuration';
+
+
+create table event_sink(
+    public_id wt_public_id primary key,
+    config_id wt_public_id 
+        constraint event_config_fkey
+            references event_config(public_id)
+            on delete cascade
+            on update cascade
+);
+comment on table event_sink is 
+'event_sink is a table where each row represents a configured event sink';
+
+create table event_file_sink(
+    public_id wt_public_id primary key,
+    sink_id wt_public_id not null
+        constraint sink_id_fkey
```

Comment:
```suggestion
        constraint event_sink_fkey
```
Our standard naming convention for foreign key constraints is the name of the foreign table with the suffix `_fkey`.